src/EventStream.js

```diff
@@ -51,15 +70,6 @@ class EventStream extends EventEmitter {
 							// don't bother doing anything special if the JSON.parse fails
 							// since we are already about to reject the promise anyway
 						} finally {
-							try {
-								this.emit('response', {
```

Comment:
`response` was emitted before the `EventStream` instance was returned to the user of `particle-api-js` so it could never be caught.